src/test/java/org/apache/sling/discovery/commons/providers/DummyTopologyView.java

```diff
@@ -199,7 +200,13 @@ public static DummyTopologyView clone(final DummyTopologyView view) {
             String clusterId = id.getClusterView().getId();
             DefaultClusterView cluster = clusters.get(clusterId);
             if (cluster==null) {
-                cluster = new DefaultClusterView(clusterId);
+                final ClusterView origCluster = id.getClusterView();
+                if (origCluster instanceof LocalClusterView) {
+                    final LocalClusterView localOrigCluster = (LocalClusterView) origCluster;
+                    cluster = new LocalClusterView(origCluster.getId(), localOrigCluster.getLocalClusterSyncTokenId());
```

Comment:
Not sure if this is relevant, but this does not copy over the `partiallyStartedClusterNodeIds`.

Reply:
It might not be relevant as this dummy test class is not used by the new code - but I think it needs to copy it as well, to make it consistent. Will do that.